inspire_utils/config.py

```diff
@@ -0,0 +1,39 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of INSPIRE.
+# Copyright (C) 2018 CERN.
+#
+# INSPIRE is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# INSPIRE is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with INSPIRE. If not, see <http://www.gnu.org/licenses/>.
+#
+# In applying this license, CERN does not waive the privileges and immunities
+# granted to it by virtue of its status as an Intergovernmental Organization
+# or submit itself to any jurisdiction.
+
+"""ORCID configuration loader."""
+
+from __future__ import absolute_import, division, print_function
+
+
+class Config(dict):
+    def __init__(self, defaults=None):
+        super(Config, self).__init__(defaults or {})
+
+    def load_pyfile(self, path):
```

Comment:
I'd add a default path, so all the places this is used can "reuse" that default one. Probably as a global in this module for now.

Reply:
Something like:
```
DEFAULT_CONF_PATH = os.path.join(os.curdir, 'inspirehep.cfg')

Reply:
:/ though currently in prod/qa the path is `os.path.join(os.curdir, 'var/inspirehep-instance/inspirehep.cfg')`.

It might be better to add a function here, like `load_config` that does that for you, something like:
```python
DEFAULT_FILE_PATHS = (
    './var/inspirehep-instance/inspirehep.cfg',
    'inspirehep.cfg',
)


def load_config(paths=DEFAULT_FILE_PATHS):
    conf = Config()
    for path in paths:
        if os.path.isfile(path):
            conf.load_pyfile(path)

    return conf
```

Or similar (not 100% convinced by the above solution though).